modules/cnf-du-partitioning-management-workloads.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-partitioning-management-workloads_{context}"]
+
+= Partitioning management workloads
+
+You can isolate the platform services, housekeeping tasks, and infrastructure pods to run on a reserved
```

Comment:
If the platform is OCP, then please use "{product-title} services".  "housekeeping tasks" is too informal--maybe just remove it.

Reply:
Replaced "housekeeping tasks" with "cluster management workloads". In this context, "infrastructure" does not have thaat meaning.